cloudbuild.dev.yaml

```diff
@@ -1,13 +1,13 @@
 steps:
-- name: 'gcr.io/cloud-builders/go:1.22'
-  args: ['version']
+- name: 'docker.io/library/golang:1.24'
```

Comment:
Only because you mentioned doing the upgrade in smaller steps, should these images be 1.23? I do trust Go's backward compatibility enough to always run latest, especially now that we have the `toolchain` directive.

Reply:
Eh, yeah, sure. That way it won't have to do the redownload of the toolchain too.